didcomm/pack_encrypted.py

```diff
@@ -79,11 +81,9 @@ async def pack_encrypted(
     :param pack_config: Configuration defining how pack needs to be done.
                         If not specified - default configuration is used.
     :param pack_params: Optional parameters for pack
-    :param resolvers_config: Optional resolvers that can override a default resolvers registered by
-                             `register_default_secrets_resolver` and `register_default_did_resolver`
 
-    :raises ValueError: If invalid input is provided. For example, if `frm` argument doesn't match `from` header in Message,
-                        or `to` argument doesn't match any of `to` header values in Message.
+    :raises DIDCommValueError: If invalid input is provided. For example, if `frm` argument doesn't match `from` header in Message,
+                               or `to` argument doesn't match any of `to` header values in Message.
```

Comment:
I believe we need to choose some docstring style ([numpy](https://numpydoc.readthedocs.io/en/latest/format.html), [google](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings), ...). But I believe for most of them one indentation block is enough for -after linebreaks:
```python
""""
    :raises DIDCommValueError: If invalid input is provided. For example, if `frm` argument
        doesn't match `from` header in Message,  or `to` argument doesn't match any of `to` header
        values in Message.
"""
```